algorithms/alg.verify-tx.tex

```diff
@@ -0,0 +1,12 @@
+\begin{algorithm}[H]
+    \caption{\label{alg.verify-tx} A function to verify that a transaction is conforming to the provided event and is included in the provided block.}
+    \begin{algorithmic}[1]
+        \Function{\sf verify-tx$_e$}{$\alpha$}
+            \If{$\neg \TxHasTransfer(a.\mathsf{tx}, e.\textsf{receivingAddress}, e.\textsf{amount})$}
```

Comment:
Let's be consistent with the text here and call this `addr` instead of `receivingAddress`